src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class PostalCode {
```

Comment:
Can implement a method to check if the postal code is in the correct format, e.g. number only.